tests/snapshots/import_system__import_main.bend.snap

```diff
@@ -3,4 +3,5 @@ source: tests/golden_tests.rs
 input_file: tests/golden_tests/import_system/import_main.bend
 ---
 [4m[1m[31mErrors:[0m
+[1mDuring execution :[0m
 Can not import the entry point of the program.
```

Comment:
Can we not report the file that originates this error? It looks strange to know where the error comes from.

Reply:
I think I'll change this so no message appears when there are only diagnostics with source.file == None, and when there are there are diagnostics with source files and ones without, the ones without will show in the end with the message Other diagnostics: or something similar. What do you think?

Reply:
Sure